designer/deployment-manager-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/DeploymentManager.scala

```diff
@@ -42,15 +42,6 @@ trait DeploymentManager extends AutoCloseable {
     Future.failed(new NotImplementedError())
 }
 
-trait ManagerSpecificScenarioActivitiesStoredByManager { self: DeploymentManager =>
```

Comment:
All activities are now stored in the single db table. DM's no longer can provide custom activities. If needed, we will be able to provide handle to save activities in the repo (as is done for scheduling, but scheduling is already in designer, so it did not require passing anything new in the DM API)